pom.xml

```diff
@@ -49,6 +49,12 @@
             <artifactId>annotations</artifactId>
             <version>23.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.24</version>
```

Comment:
I worry about getting a dependency mismatch/conflict with whatever Engine uses. I couldn't find it for master now stuff's been changed. Is it possible to not specify version?

Reply:
I think so. Thanks, I'll remove the version.

Reply:
Argh maven fails when the version value isn't there.

Reply:
It's OK, I'll exclude lombok from jnode-fs in engine.